### PR TITLE
[B2BP-337] Added Update Static Content plugin for static site deployment

### PR DESCRIPTION
**Pull request by simosalsi**
Added Update Static Content plugin for static site deployment.
Depends on #142 

[RFC](https://pagopa.atlassian.net/wiki/spaces/SV/pages/903151626/SV-014+Avviare+il+re+deploy+del+sito+statico+da+Strapi) for reference.

#### List of Changes

- Installed plugin
- Configured plugin
- Updated .env.example

#### Motivation and Context

Needed to give CMS users the chance to trigger redeployment.

#### How Has This Been Tested?

Inside a production-equivalent AWS environment.

#### Screenshots (if appropriate):
Screenshot (19)

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by simosalsi** [comment]
@maurodandrea 

We may need a new PR to add the environment variables needed for this to work.

Namely the following:
GITHUB_PAT
REPO_OWNER
REPO_NAME
WORKFLOW_ID
TARGET_BRANCH

All except for GITHUB_PAT are currently optional, as they have a default value.

However we may not need some of these at all: If the above mentioned "default values" are sure to be the only possible values we may as well hard-code them in.

**Comment by maurodandrea** [comment]
Depends on https://github.com/pagopa/b2b-portals/pull/142

File: .changeset/spicy-bats-wash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"strapi-cms": patch
+---
+
+Added Update Static Content plugin for static site deployment
```
Comment (datalek) [inline .changeset/spicy-bats-wash.md:5]:
```suggestion
"strapi-cms": minor
---

Add Update Static Content plugin for static site deployment
```

**Review by datalek — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]
👍

**Review by kin0992 — APPROVED** [review]
👍

**Review by maurodandrea — APPROVED** [review]

**Review by datalek — APPROVED** [review]

**Review by datalek — APPROVED** [review]

**Pull request merged**